app/_hub/kong-inc/response-transformer/index.md

```diff
@@ -69,44 +65,56 @@ params:
     - name: remove.headers
       required: false
       value_in_examples: ["x-toremove", "x-another-one"]
+      datatype: array of string elements
       description: List of header names. Unset the header(s) with the given name.
     - name: remove.json
       required: false
       value_in_examples: ["json-key-toremove", "another-json-key"]
+      datatype: array of string elements
       description: List of property names. Remove the property from the JSON body if it is present.
     - name: rename.headers
       required: false
-      description: List of original_header_name:new_header_name pairs. If the header `original_headername` is already set, rename it to `new_headername`. Ignored if the header is not already set.
+      datatype: array of string elements
+      description: List of `original_header_name:new_header_name` pairs. If the header `original_headername` is already set, rename it to `new_headername`. Ignored if the header is not already set.
     - name: replace.headers
       required: false
-      description: List of headername:value pairs. If and only if the header is already set, replace its old value with the new one. Ignored if the header is not already set.
+      datatype: array of string elements
+      description: List of `headername:value` pairs. If and only if the header is already set, replace its old value with the new one. Ignored if the header is not already set.
     - name: replace.json
       required: false
-      description: List of property:value pairs. If and only if the parameter is already present, replace its old value with the new one. Ignored if the parameter is not already present.
+      datatype: array of string elements
+      description: List of `property:value` pairs. If and only if the parameter is already present, replace its old value with the new one. Ignored if the parameter is not already present.
     - name: replace.json_types
       required: false
+      datatype: array of string, boolean, or number elements
```

Comment:
this is also an `array of string elements`, where each string can be either one of: boolean, number, string

relates to my previous question about the `one_of` representation

Reply:
Done in commit 4217372.

We shouldn't say "either" because that means two choices, not three.